CONTRIBUTING.md

```diff
@@ -1,22 +1,49 @@
+
+Issues and PRs in the ecma262 repository are appropriate for minor modifications to the existing specification, for example to make behavior more consistent in an edge case, match what is implemented by  the large majority of actual implementations, clarify wording, or occasionally add a minor feature. For bigger changes, use the feature request process described below.
+
+#### New feature proposals
+TC39 is open to accepting new feature requests for ECMAScript, referred to as "proposals". Proposals go through a four-stage process which is documented in the [TC39 process document](https://tc39.github.io/process-document/).
+
+Feature requests for future versions of ECMAScript should not be made in this repository. Instead, they are developed in separate GitHub repositories, which are then merged into the main repository once they have received "Stage 4".
 
-#### Bug Reports & Bug Fixes
-File a bug or pull requests against the current text of ECMA262 in this repository. If the fix is trivial you may not need to sign the CLA. If your fix is involved, signing the CLA will be required.
+##### Creating a new proposal
+To make a feature request, document the problem and a sketch of the solution with others in the community, including TC39 members. One place to do this is the [es-discuss mailing list](http://esdiscuss.org); another is #tc39 on Freenode ([instructions](https://freenode.net/kb/answer/chat)).
 
-#### Feature Requests
-Feature requests for future versions of ECMAScript should not be made in this repository. To make a feature request, post to the [es-discuss mailing list](http://esdiscuss.org). Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language.
+Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language. Once a proposal is introduced to the committee, new features are considered by the committee according to the [TC39 process document](https://tc39.github.io/process-document/).
 
-#### Proposals
-If you have a new proposal you want to get into the language, you first need a TC39 champion. Head to [es-discuss](http://esdiscuss.org) to find one. If you already have a TC39 champion, send a pull request to add a link to your proposal to the Current Proposals list or the stage zero list.
+##### TC39 meetings and champions
+If you have a new proposal you want to get into the language, you first need a TC39 "champion": a member of the committee who will make the case for the proposal at physical TC39 meetings and help it move through the process. If you are a TC39 member, you can be a champion; otherwise, find a TC39 member to work with for this (e.g., through es-discuss or #tc39). Proposals may have multiple champions (a "champion group").
```

Comment:
s/physical/in-person. ~~Could also link to the agendas repo here.~~ actually it doesn't have the meeting schedule, maybe here instead: https://www.ecma-international.org/memento/TC39-M.htm